ui/src/testing/stubs/stub-utils.tsx

```diff
@@ -0,0 +1 @@
+export const StubImplementationRequired = Error('Cannot fetch in tests.  Please implement stub method.');
```

Comment:
This is named like a class, but it's actually a constant, please lower case.

Also, maybe put Error in the name: `stubNotImplementedError`, ?